idaes/core/initialization/initialize_from_data.py

```diff
@@ -0,0 +1,24 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Initializer class for implementing initialization from a data source
+"""
+from idaes.core.initialization.initializer_base import InitializerBase
+
+__author__ = "Andrew Lee"
+
+
+class FromDataInitializer(InitializerBase):
+    def initialization_routine(self, model):
+        # No action required
+        pass
```

Comment:
Is the assumption that something else already initialized the model?  Then I would rename this to the "BypassInitializer" or the "NoOpInitializer"  A "FromData" should take a data source...